packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -76,9 +87,28 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     onRenderAvatar,
     showMessageStatus,
     messageStatus,
-    fileDownloadHandler
+    fileDownloadHandler,
+    /* @conditional-compile-remove(dateTimeCustomization) */
+    onDisplayDateTimeString
   } = props;
 
+  // onDisplayDateTimeString from messagethread overwrites onDisplayDateTimeString from locale overwrites default
```

Comment:
This logic is hard to follow. Can we simplify it by doing something like this?
![image](https://user-images.githubusercontent.com/9782747/169588351-292c4385-fc28-4202-b337-3bfa95e96e87.png)